rust/worker/src/blockstore/arrow/block/mod.rs

```diff
@@ -11,4 +11,5 @@ mod types;
 mod u32_key;
 mod u32_value;
 // Re-export types at the arrow_blockfile module level
-pub(in crate::blockstore::arrow) use types::*;
+// pub(in crate::blockstore::arrow) use types::*;
```

Comment:
why is this commented out? Do we need to export these types outside the blockfile level?